dev-util/cgdb/cgdb-0.7.0-r2.ebuild

```diff
@@ -40,6 +40,8 @@ DOCS=( AUTHORS ChangeLog FAQ INSTALL NEWS README.md )
 PATCHES=(
```

Comment:
Also, is there a real benefit in having multilib here?

Reply:
Hm, not sure really, do you think it should be removed?

Reply:
I think so, but we don't have to do it this time. Just for next time around. It removes some complexity from the ebuild.